ext/afform/admin/ang/afGuiEditor/afGuiSearch.html

```diff
@@ -19,7 +19,7 @@
               <a href ng-click="filter.mode = 'val'; filter.value = ''">{{:: ts('Fixed value') }}</a>
             </li>
             <li ng-if="$ctrl.editor.isContactSummary()">
-              <a href ng-click="filter.mode = 'options'; filter.value = 'contact_id';">{{:: ts('Current Contact') }}</a>
+              <a href ng-click="filter.mode = 'options'; filter.value = 'entity_id';">{{:: ts('Contact being Viewed') }}</a>
```

Comment:
Minor thing: I think L11 above also needs to switch from Current Contact . Otherwise there's a mismatch between the select options and the display:

![image](https://github.com/user-attachments/assets/930e9a0b-193c-4831-ad33-b3a2c8b8e811)

Reply:
Good catch. Fixed.
FYI the rename was because I'm adding a similar option for submission forms so that they too can be placed on the contact summary page in-context, and I wanted it to have a label that wasn't so similar to "Current User"

Reply:
Yep I think it's a good change - have found the "Current Contact" a bit confusing here before